src/Interfaces/InitializationSchemeInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Kmeans\Interfaces;
+
+use Kmeans\Interfaces\ClusterCollectionInterface;
+use Kmeans\Interfaces\PointCollectionInterface;
```

Comment:
ClusterCollectionInterface and PointCollectionInterface are the same namespace as InitializationSchemeInterface.
Therefore, "use Kmeans\Interfaces\ClusterCollectionInterface" and "use Kmeans\Interfaces\PointCollectionInterface" can be deleted unless they are intended to be used explicitly.